File: Android/app/src/main/java/com/unicef/mis/enumtype/CollectionStatus.java
package com.unicef.mis.enumtype;

public enum CollectionStatus {
    NoStatus(0),
    NotCollected(1),
    Collected(2),
    Approved(3),
    Requested_Inactive(6),
    Recollect(4);

    private int intValue;

    CollectionStatus(int intValue) {
        this.intValue = intValue;
    }

    public int getIntValue() {
        return intValue;
    }

    public static CollectionStatus fromInt(int intValue){
        for (CollectionStatus status : CollectionStatus.values()) {
            if (status.getIntValue() == intValue) { return status; }
        }
        return null;
    }
}

File: Api/src/UnicefEducationMIS.Core/ValueObjects/CollectionStatus.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace UnicefEducationMIS.Core.ValueObjects
{
    public enum CollectionStatus
    {
        NotCollected = 1,
        Collected = 2,
        Approved = 3,
        Recollect = 4,

        Deleted = 5,
        Requested_Inactive = 6,
        Inactivated = 7

    }
}
